### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by baratamas**
#2 
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by jancschaefer — COMMENTED** [review]
Are you serious?????

File: test2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test2
```
Comment (jancschaefer) [inline test2.txt:1]:
```suggestion
test2 
test2.2
```

**Review by umtdgmz — COMMENTED** [review]
great work

**Review by ozgurozturknet — APPROVED** [review]
good job

**Comment by jancschaefer** [comment]
amazing!

**Review by umtdgmz — APPROVED** [review]

**Review by jancschaefer — CHANGES_REQUESTED** [review]

**Review by jancschaefer — APPROVED** [review]

**Pull request merged**